### ISSUE TITLE
Encoder problem

### ISSUE THREAD

**Issue by racalac** [paulsammut/ramosclock#2]
Hi, do you have a part number for the encoder? Mine is acting up after all this time. I rotate to change the alarm time and 2 clicks do nothing, next click changes the value by 1, next click does nothing, next click lowers the value by 1. Probably something impeding part of the quadrature encoding. Blowing air into the part fixed things temporarily.

A BOM would be very helpful for repair work.

Thank you.

**Comment by MiskatonicU** [comment]
@racalac Don't suppose you ever got any further details form @paulsammut ?  My LED Ramos started having this same issue and I was hoping it would just be a matter of replacing the encoder, but that means knowing the part number.

**Comment by racalac** [comment]
@MiskatonicU Sorry I never heard from him. Maybe a replacement part is obvious to someone who works with them more often?

**Comment by paulsammut** [comment]
Sorry i missed this guys!!! 

Here's the part: https://www.digikey.com/en/products/detail/cts-electrocomponents/288X232R161B2/270037

It's got a 3 wire cable soldered to it terminated to 3 pin MTA100 connector. Would be easy to desolder and solder it back on the replacement. Just keep note of the pinout to wire color.

FYI CAD is here which is where i found the part number cause I lost all the bom files this project 😂

https://grabcad.com/library/nixie-ramos-alarm-clock-1/details?folder_id=5791349

OH and we cut the rod of the encoder! I forget the measurement but we trimmed it to fit the knob. You can put it in a vise and cut it down with a hacksaw.

**Comment by racalac** [comment]
Wow, thank you! Is it the same part as in the LED Ramos? Oh my god were you hand cutting all the parts when they were in production? 

Thanks again, I love my Ramos and hope to keep it getting my butt out of bed for years to come. Really appreciate posting all the tech details long after the end of the project!

**Comment by paulsammut** [comment]
Yup same one in the LED ramos. Hah yeah cut em by hand but we had a lathe at least.

I may have some spares (cut and with cable) laying around - I'll check this week. DM your address and I'll send it over when/if i find it

**Comment by paulsammut** [comment]
Yup same one in the LED ramos. Hah yeah cut em by hand but we had a lathe at least.

I may have some spares (cut and with cable) laying around - I'll check this week. DM your address and I'll send it over when/if i find it.

Thanks for buying this guy from us and happy it's been working all these years!!

**Comment by MiskatonicU** [comment]
@paulsammut You already got in contact with me via email, but I'm glad you also responded here as I saw someone else had posted with the same issue.

Also, sorry again for pinging you on so many platforms,

**Comment by racalac** [comment]
> I may have some spares (cut and with cable) laying around - I'll check this week. DM your address and I'll send it over when/if i find it.
> 
> Thanks for buying this guy from us and happy it's been working all these years!!

Thank you very much for the part offer. I'm more of a software type than a hacksaw type. What platform or address for the DM?

**Comment by paulsammut** [comment]
Ah sorry guys just checked my boxes of clock parts and I dont have spares. you can order a replacement from Digikey and desolder the wires and solder on the new one. Also you'll have to cut the stem to the same height.

Lmk if u need more info on how to do it!

**Comment by MiskatonicU** [comment]
@paulsammut 

No worries, I appreciate that you checked if you had any replacements.

I too am more of a software person than hardware.  I've used an iron to do some through-hold soldering, but it's been quite some time and I don't own an iron anymore.

I roughly described what I understand to be the steps involved in the repair to an engineer friend and he thought the soldering work would be easy enough and offered to do it for me if can provide the parts.

I haven't opened the clock myself yet, but I took a few captures from this [YouTube video where the backup battery was replaced](https://www.youtube.com/watch?v=4GUXdHicJHw) to get an idea of the internals.

If you don't mind, could you give a rough set of steps involved with the repair?  If that's a bit too involved, maybe just answering the following questions will clear up a few of the steps for me.

1. What is needed to prepare the new encoder for installation?  It sounds like the stem will need to be cut down to the correct height.  Not sure I can get access to a belt sander, so maybe a vice and hacksaw will do?  

2. In the above video,  a screwdriver was ground down to fit the screws used to secure the clock's base panel.  Is there a particular size and style of bit to use to avoid the need to sacrifice a screwdriver?
 
3. I think the biggest unknown for me is how to remove the existing encoder from the clock, as I'm not entirely sure how it's secured to the top of the clock.  I assume the cap can be removed just by loosening the screw securing it to the stem, but how is the base of the encoder mounted inside?  Should the internal battery be disconnected before removing the encoder?

4. The other end of the wires terminates in a 3 pin connector which plugs in somewhere on the main board, does the orientation matter?  Any chance you can attach a picture of the main board highlighting the pins the connector goes to?

5. Any other repairs I should make while I have the clock open?  Replace the CR2032 battery?  Replace the backup battery as in the Youtube video?  If the latter is recommended, is searching eBay for "UB1208K" still the best way to get a replacement?

**Comment by paulsammut** [comment]
1. cut the encoder shaft same length as the orig. Chuck the waste stem end and cut with a hacksaw
2. use a square drive bit
3. take the knob off and use a socket to loosen the nut that holds the encoder
4. take a pick of where the encoder connector is on the main board and put it in the same exact spot
5. yup both those things are good to do

**Comment by MiskatonicU** [comment]
@paulsammut Thanks again for the details.  I just got the replacement encoder, waiting on the battery and should be able attempt the repair soon.

**Comment by MiskatonicU** [comment]
@paulsammut @racalac 
Took longer than I would have liked but my clock is working again, though it appears my original issue wasn't the encoder itself...

The replacement encoder matched the original and after I had a friend trim and solder it to the daughter board I was able to reinstall the assembly without issue.  Unfortunately, my problem persisted.

I took the assembly back out and inspected it closely, and noticed one of the leads from the harness to the daughter board seemed very tenuous and the solder joint broke when I pulled the wire a bit.  Guessing this broken connection was actually the source of my problem I reinstalled the assembly and confirmed the problem was still the same as before, so I feel pretty confident this weak (and now broken) connection was the source of my problem.

My friend was able to trim and solder the wire back onto the pad on the daughter board, after a quick test install, I confirmed the encoder assembly is now working again, and no longer gets stuck cycling between the same digits.

The old encoder was destroyed in the process of installing the new one (old leads cut off to separate it form the daughter board), so I can't say for sure it would have worked if the wire had been fixed, but the new one does work once ensuring all the leads to the daughter board were solid.

My clock is now fully assembled again and working.  I also took this chance to replace both the lead acid battery (which was very swollen) and the coin battery inside.  Hoping the clock will now last another 10+ years before it needs servicing again.

**Comment by paulsammut** [comment]
Ah loose connection strikes again! Glad you were able to solve it.

**Comment by Ramosuser007** [comment]
Hi. The 5 cable soldering has come apart on my ramos adjuster knob. Can anyone assist with a wiring diagram so I can try reconnecting? @paulsammut 
I'm lost without my 11 year old alarm clock... only thing that was ever worked. Best invention since sliced bread💪 
Many thanks,  Mark

**Comment by paulsammut** [comment]
This should help:


schematics are in the repo. Not sure which blue wire goes where but it’s
50/50 so give it a shot :)

**Comment by paulsammut** [comment]
This should help:

![image](https://github.com/user-attachments/assets/040ab4f2-8a63-4ddd-99d1-64d2e70ea1e6)

schematics are in the repo. Not sure which blue wire goes where but it’s 50/50 so give it a shot :)

**Comment by Ramosuser007** [comment]
> This should help:
> 
> ![image](https://github.com/user-attachments/assets/040ab4f2-8a63-4ddd-99d1-64d2e70ea1e6)
> 
> schematics are in the repo. Not sure which blue wire goes where but it’s 50/50 so give it a shot :)

Thanks Paul. Thats magic! You're a gentleman.
